www/Makefile

```diff
@@ -1,9 +1,11 @@
-build:
-	@npm install
+public/script.js: package-lock.json $(wildcard src/*.js)
 	@npm run build
 	@awk 1 build/static/js/*.js > public/script.js
 
+package-lock.json:
+	@npm install
```

Comment:
Ideally there would be a package-lock.json file,
and then this command would be `@npm ci`
to install a reproducible build from tested source.

Reply:
Seems reasonable